backup-stores/s3/src/main/java/io/camunda/zeebe/backup/s3/S3BackupStore.java

```diff
@@ -343,7 +355,18 @@ private CompletableFuture<PutObjectResponse> saveNamedFile(
         AsyncRequestBody.fromFile(filePath));
   }
 
-  public static String objectPrefix(BackupIdentifier id) {
-    return "%s/%s/%s/".formatted(id.partitionId(), id.checkpointId(), id.nodeId());
+  private static S3AsyncClient buildClient(S3BackupConfig config) {
```

Comment:
```suggestion
  private static S3AsyncClient buildClient(final S3BackupConfig config) {
```

Reply:
Huh, there are a bunch of missing finals. I thought these were added automatically by IntelliJ :thinking: